### PR TITLE
Invoked Downshift closeMenu function after selecting dates

### PR DESCRIPTION
**Pull request by dreamsparkx**
Invoked Downshift closeMenu function after selecting dates

### CODE REVIEW

File: lib/src/DateRangePicker.js [hunk 1]
```diff
@@ -67,6 +67,7 @@ class DateRangePicker extends PureComponent {
     // is given and the selected date is after the start date
     if (fromDate && isAfter(selectedDate, fromDate)) {
       this.props.onChange({ toDate: selectedDate });
+        stateAndHelpers.closeMenu();
```
Comment (dreamsparkx) [inline lib/src/DateRangePicker.js]: Called `closeMenu()` function.

**Comment by jonas-app** [comment]
Thx for your contribution, @dreamsparkx 🥇 
Sorry for the late response, was on vacation.
Can you make this behaviour optional and configurable via props?

**Comment by dreamsparkx** [comment]
Added behaviour via props.

**Comment by jonas-app** [comment]
@dreamsparkx Thx again for your contribution. 👏 
I changed it a little bit to make it work in all cases.
For example if both dates where already selected and you select a new from date.

I released [version 0.1.1](https://www.npmjs.com/package/material-date-range-picker/v/0.1.1) to npm with this feature.

**Comment by jonas-app** [comment]
Closes #7

**Pull request merged**